### PR TITLE
New Allowed chars in M+

### PR DESCRIPTION
**Pull request by wushin**

### CODE REVIEW

**Comment by wushin** [comment]
@4144 any you can see left that shouldn't be in?

File: world/conf/char_athena.conf [hunk 1]
```diff
@@ -28,21 +28,17 @@ char_log_filename: log/char.log
...
-//char_name_letters: []
-char_name_letters: @\^_`
+char_name_letters: \^_`
 char_name_letters: abcdefghijklmnoprstquvwxyz
-//char_name_letters: {}
 char_name_letters: |~
 
 // minimum char name length
-min_name_length: 4
+min_name_length: 5
```
Comment (mekolat) [inline world/conf/char_athena.conf]: why has the min length been increased?

Comment (4144) [inline world/conf/char_athena.conf]: min len is 4. wushin probably you not understood code. server message in format: "nick : message" part "nick " must be at least 5 chars, and nick here at least 4 chars (without space)

Comment (mekolat) [inline world/conf/char_athena.conf]: ah I see so you count the spaces instead of trimming

Comment (wushin) [inline world/conf/char_athena.conf]: Aye, I get it.

**Pull request merged**